debezium-server/debezium-server-redis/src/main/java/io/debezium/server/redis/RedisStreamChangeConsumer.java

```diff
@@ -129,9 +157,13 @@ public void handleBatch(List<ChangeEvent<Object, Object>> records,
             throws InterruptedException {
         DelayStrategy delayStrategy = DelayStrategy.exponential(initialRetryDelay, maxRetryDelay);
 
-        LOGGER.info("Handling a batch of {} records", records.size());
+        LOGGER.trace("Handling a batch of {} records", records.size());
         batches(records, batchSize).forEach(batch -> {
             boolean completedSuccessfully = false;
+            if (relativeMinIdTrim != null && maxLenTrim == null) {
+                long minIdTrim = System.currentTimeMillis() - relativeMinIdTrim;
```

Comment:
Let's rather use `io.debezium.util.Clock`, stored in a field, initialized in the constructor:

    this.clock = Clock.system();

That being said, I'm confused by the semantics: as per [XTRIM docs](https://redis.io/commands/xtrim#nearly-exact-trimming), this is about an id, apparently sourced from some kind of sequence. How can this be subject to an operation based on epoch milliseconds?